tests/integration/test_rc_kickoff.py

```
@@ -0,0 +1,171 @@
+        )
+    finally:
+        # Cleanup discipline mirrors the guest-exec'd rc tests' reliance on
+        # test_shed_name_dev's shed-level teardown, but ALSO exercises (and proves)
+        # the CLI's own session-kill path so a failed run doesn't leak a live rc-*
+        # tmux session for the lifetime of the dev shed. Best-effort: a failure here
+        # must never mask the real assertion failure above.
+        if slug is not None:
+            try:
+                server.cli("sessions", "kill", shed, f"rc-{slug}", timeout=30)
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)